content/agent/kubernetes/daemonset_setup.md

```diff
@@ -0,0 +1,305 @@
+
+### Verification
+
+To verify the Datadog Agent is running in your environment as a DaemonSet, execute:
+
+    kubectl get daemonset
+
+If the Agent is deployed, you will see output similar to the text below, where `DESIRED` and `CURRENT` are equal to the number of nodes running in your cluster.
+
+    NAME            DESIRED   CURRENT   READY     UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+    datadog-agent   2         2         2         2            2           <none>          16h
+
+
+## Enable capabilities
+
+### Log Collection
+
+To enable [Log collection][10] with your DaemonSet:
+
+1. Set the `DD_LOGS_ENABLED` and `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` variable to true in your *env* section:
+
+    ```
+    (...)
+      env:
+        (...)
+        - name: DD_LOGS_ENABLED
+            value: "true"
+        - name: DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL
+            value: "true"
+    (...)
+    ```
+
+2. Mount the `pointdir` volume in *volumeMounts*:
+
+    ```
+      (...)
+        volumeMounts:
+          (...)
+          - name: pointerdir
+              mountPath: /opt/datadog-agent/run
+      (...)
+      volumes:
+        (...)
+        - hostPath:
+              path: /opt/datadog-agent/run
+            name: pointerdir
+      (...)
+    ```
+
+Learn more about this in [the Docker log collection documentation][11].
+
+### Trace Collection
+
+To enable [Trace collection][20] with your DaemonSet:
+
+1. Set the `DD_APM_ENABLED` variable to true in your *env* section:
+
+    ```
+    (...)
+      env:
+        (...)
+        - name: DD_APM_ENABLED
+            value: "true"
+    (...)
+    ```
+
+2. Uncomment the `# hostPort: 8126` line. 
+  This exposes the Datadog Agent tracing port on each of your Kubernetes nodes. 
+
+    **Warning**: This opens a port on your host. Make sure your firewall covers that correctly. 
+    Another word of caution: some network plugging don't support `hostPorts` yet, so this won't work. The workaround in this case is to add `hostNetwork: true` in your agent pod specifications. This shares the network namespace of your host with the Datadog agent. Again, make sure this logic is okay with your security policies.
+
+### Process Collection
+
+See [Process collection for Kubernetes][21].
+
+### DogStatsD
+
+To send custom metrics via DogStatsD, set the `DD_DOGSTATSD_NON_LOCAL_TRAFFIC`variable to true in your *env* section:
+
+```
+(...)
+      env:
+        (...)
+        - name: DD_DOGSTATSD_NON_LOCAL_TRAFFIC
+          value: "true"
+(...)
+```
+
+Learn more about this in the [Docker DogStatsD documentation][19]
+
+To send custom metrics via DogStatsD from your application pods, uncomment the `# hostPort: 8125` line in your `datadog-agent.yaml` manifest. This exposes the DogStatsD port on each of your Kubernetes nodes. 
+
+**Warning**: This opens a port on your host. Make sure your firewall covers that correctly. 
```

Comment:
See previous comments on this.

Also, rather than repeating this same text, should we just have one section at the bottom called Security that points out both scenarios and has the text once? I do realize that doing so might mean people stop reading after the trace collection section and miss it, but it's worth discussing.